mandatory/1-fix-functions.js

```diff
@@ -10,43 +10,38 @@
 
 */
 
-function mood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function mood(isHappy) {
+  if (isHappy == true) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough = num;
```

Comment:
we don't normally assign a variable into another variable without doing any manipulation. Can you move the condition into `isBigEnough` variable?